src/common/components/cards/cards-visualization-modifier-buttons.tsx

```diff
@@ -1,57 +1,38 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import {
-    cardsVisualizationModifiersContainer,
-    expandCollapseAllButton,
-    visualHelperToggle,
-} from 'common/components/cards/cards-visualization-modifiers-buttons.scss';
-import { CardSelectionMessageCreator } from 'common/message-creators/card-selection-message-creator';
-import { NamedFC } from 'common/react/named-fc';
-import { ActionButton, Toggle } from 'office-ui-fabric-react';
+    ExpandCollapseAllButton,
+    ExpandCollapseAllButtonProps,
+} from 'common/components/cards/expand-collapse-all-button';
+import {
+    VisualHelperToggle,
+    VisualHelperToggleProps,
+} from 'common/components/cards/visual-helper-toggle';
+import { NamedFC, ReactFCWithDisplayName } from 'common/react/named-fc';
 import * as React from 'react';
+import * as styles from './cards-visualization-modifier-buttons.scss';
 
-export type CardsVisualizationModifierButtonsDeps = {
-    cardSelectionMessageCreator: CardSelectionMessageCreator;
-};
-
-export type CardsVisualizationModifierButtonsProps = {
-    deps: CardsVisualizationModifierButtonsDeps;
-    visualHelperEnabled: boolean;
-    allCardsCollapsed: boolean;
-};
+export type CardsVisualizationModifierButtonsProps = ExpandCollapseAllButtonProps &
+    VisualHelperToggleProps;
 
-export const CardsVisualizationModifierButtons = NamedFC<CardsVisualizationModifierButtonsProps>(
-    'CardsVisualizationModifierButtons',
+export const ExpandCollapseOnlyModifierButtons = NamedFC<CardsVisualizationModifierButtonsProps>(
```

Comment:
Should the file name be updated as well?

Reply:
I left the filename as-is (matching the shared props type) because this file intentionally exports 2 different variants of `CardsVisualizaitonModifierButton`; maybe exporting a type by that name that both impls adhere to would be more clear?

Reply:
Added a commit doing this